woocommerce/woocommerce-ios
# Remove IPP Feedback banner

Closes: #11276
Apologies for the large PR, ~500 lines of it belongs to test removal and cannot be done on a separate PR.

## Description
This PR removes the IPP feedback banner from the Order view which was shown to merchants based on different scenarios. This was a temporary feature to gather feedback, and now it's a good time to remove it:

> We implemented it with the plan to remove it a couple months later. The solution uses several workarounds and is far from perfect. Since it’s also the only place where we use receipt_url order metadata which we might want to remove from the backend, I think removing its usage in the apps would be a good first step.

## Changes
- Removed logic from `OrderListViewModel` and `OrderListViewController`, so the banner is only used for other feedback types (order creation, and displaying errors)
- Removed track events and survey links to the different IPP merchants categories (CashOnDeliveryFeedback, FirstTransactionFeedback, PowerUsersFeedback)
- Updated Unit Tests

## Testing instructions
1. On a fresh store (I believe this is the only requirement), confirm that the Order creation banner still appears and works (screenshot 1). On an unrelated note, it might also be a good time to remove this one. Opened discussion here: p91TBi-aPz-p2
2. On any store, with connectivity off, confirm that the error banner still appears and works (screenshot 2)

## Screenshots
| Order creation banner | Error banner |
|--------|--------|
| <img width="800" src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/356a13a1-8bb9-439b-8d32-477ba9cd9db3"> | <img width=800 src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/7aead7c6-01f7-412b-948d-8a67d2fea958"> | 

---
